### ISSUE TITLE
Invalid data (INF, -INF or NAN). in bufferview [5]:"for accessor [5]:"

### ISSUE THREAD

**Issue by azrsim** [AsoboStudio/glTF-Blender-IO-MSFS#331]
### Current Behavior

I get this dynamic error when I export models to MSFS, its an error thats getting on my nerves now. 

Models that are already in the simulator exported from blender get this error dynamically, as in lets say i hit clean all the error occurs. Or when I try to re-export a model the error occurs. I've done everything in blender before exporting the model according to every tutorial (mesh cleanups, apply transforms, normal maps check) everything.

Its not a blender related issue, but its moreover a issue related to the exporter and its hard to identify what the error means and what it wants.
![afbeelding_2024-08-15_131552197](https://github.com/user-attachments/assets/7d0d763d-a6d6-4a90-aaa7-4f028b91cbc9)

### Expected Behavior

That the model should load in or that I know what causes the error 

### Steps To Reproduce

export from blender

### Environment

```markdown
- OS:
- Blender: 3.6.5
- glTF-Blender-IO-MSFS:
```

**Comment by azrsim** [comment]
For people facing this issue or other bugs that are not browseable, I highly recommend them to run modelconverterx and import the exported model from blender (from the msfs gltf exporter) and re-export them via modelconverterx 

This should fix it, the msfs blender exporter is buggy

**Comment by SirComand** [comment]
I disagree with you azrsim. There are problems that can be solved and exporting to modelconvertx depending on what you export from it is exported incomplete as for example the vertex color. The transparency of the decals you do for vertex color.

**Comment by SirComand** [comment]
That error happened to me once and I fixed it but I don't remember what I did. I think it is due to a bug in the way the mesh model is exported, but it is due to a problem with the GLTF addon that blender comes with.

**Comment by SirComand** [comment]
I don't know why they don't make their own GLTF addon for this MSFS addon because this addon is not made to export custom parts needed by the simulator.

**Comment by azrsim** [comment]
Its related to your export settings, if modelconverterX doesnt load your model in then you have something done wrong.

If your model loads in modelconverterx fine in but gives a error in MSFS then its related to the exporter. Because models that are corrupt according to MSFS its SDK should not even load in modelconverterx.

**Comment by SirComand** [comment]
I understand that, I'm just saying that there are errors that occur because of things that you don't know how they work. I mean at least many times it is better to select the Collection of the Scene the one that is on the top and export it. You don't know the problems that I have solved doing that. Now I don't know why it works.

Another error that has no solution and is random is Exporting scenes with parallax. It can only be fixed by exporting it several times until in one of all exports it exports well.

**Comment by SirComand** [comment]
Also happens if you copy an object between files that the material if you have a material already created with MSFS material this makes it corrupt is always better if you copy objects between files is to reset the material by selecting again the same material ie if you had Standard Material re-select Standard Material. I know the solution to several problems but some are so sporadic that I forget the solution to them.

**Comment by y-khodja** [comment]
> I don't know why they don't make their own GLTF addon for this MSFS addon because this addon is not made to export custom parts needed by the simulator.

The blender exporter is updated with the different versions of Blender. We want to stick to LTS versions to keep the best version of the official glTF exporter. That way, we only need to plug our plugin to the right moment to export different extensions. Don't hesitate to report your issues with the khronos exporter directly on their github. Every issue matters.

Thank you

**Comment by SirComand** [comment]
> > I don't know why they don't make their own GLTF addon for this MSFS addon because this addon is not made to export custom parts needed by the simulator.
> 
> The blender exporter is updated with the different versions of Blender. We want to stick to LTS versions to keep the best version of the official glTF exporter. That way, we only need to plug our plugin to the right moment to export different extensions. Don't hesitate to report your issues with the khronos exporter directly on their github. Every issue matters.
> 
> Thank you

thank you very much then I will keep reporting those problems to be solved. Thank you very much for the reply.

**Comment by Losco-Nosciuto** [comment]
> Its related to your export settings, if modelconverterX doesnt load your model in then you have something done wrong.
> 
> If your model loads in modelconverterx fine in but gives a error in MSFS then its related to the exporter. Because models that are corrupt according to MSFS its SDK should not even load in modelconverterx.

That's as simple as you just said and works the other way around.
If you export and get the error means that something has been done wrong since all models work, and we always used it :) There's bug, but nothing that can't be worked around. That error never happened in over 300 models I made, so means that probably your model has some issues geometry or texture wise, or either something got bugged in the settings. Surely there's a bug causing it somewhere where a gltf is exported with extra data or less data, but again, 90% of the time it works, so just figure out the issue, instead of relying on a third party which is not officially supported for exporting and importing in-sim :)
